src/api/localSettings.ts

```diff
@@ -91,7 +91,7 @@ async function homepageLocaleSettingsAdapter(
             href: '/map/observation/landscape',
             images: [
               {
-                thumbnail: '/medias/placeholder3.jpg',
+                thumbnail: '/medias/placeholder4.jpg',
```

Comment:
why this changed ? there is no placeholder4.jpg file in /public/medias folder

Reply:
C'est moi, c'est pas la bonne photo qui est appelé sur la page d'accueil pour les éléments paysagers : 
![image](https://github.com/Georiviere/Georiviere-public/assets/19687397/95da173e-1936-4892-89ee-5eeacfb4ed1f)

Reply:
@thomaspnrhj en effet nous étions dans l'attente de ton retour pour nous communiquer des descriptions et images à mettre par défaut. 

Si tu fais cette modification il faut aussi ajouter une image placeholder4 dans le dossier `public/medias`